### ISSUE TITLE
install_requires line is wrong: enum34 is only available for python versions <3.4

### ISSUE THREAD

**Issue by yurivict** [Maratyszcza/PeachPy#95]
https://github.com/Maratyszcza/PeachPy/blob/master/setup.py#L91

**Comment by Maratyszcza** [comment]
Does it break anything on Python >= 3.4?

**Comment by yurivict** [comment]
Package dependency line can't include enum34 when python version is >=3.4. These dependency lines usually correspond to what is in ```setup.py``` but in this case there's a discrepancy. There's syntax to specify such version-specific dependencies in ```setup.py```.

**Comment by Maratyszcza** [comment]
As you are more knowledgeable about it than me, maybe you could submit a pull request?

**Comment by yurivict** [comment]
The correct line is:
```
':python_version == "2.7" or python_version == "3.3"': ['enum34>=1.0.4, <2']
```

**Comment by Maratyszcza** [comment]
Fixed in 5fb95df0f380773b8c079e920bd46b24e3cf5e44
